src/replaceCustomValues.ts

```diff
@@ -215,6 +265,10 @@ export function getElementSubmissionValue({
       const value = unknown as string[]
       const selectedOptionLabels: string[] = value.reduce(
         (labels: string[], selectedOption: string) => {
+          //TS making me do this again
+          if (formElement?.type !== 'checkboxes') {
```

Comment:
If you define `const options = ...` outside of the reduce you won't have to do this.